dvc/fs/hdfs.py

```diff
@@ -100,14 +102,15 @@ def close(self):
             path_info.host,
             path_info.port,
             user=path_info.user,
+            kerb_ticket=kerb_ticket,
```

Comment:
Seems like you should be able to simply
```suggestion
            kerb_ticket=self.kerb_ticket,
```
and avoid the rest of the logic. Or does something break because of it?

Reply:
If you are worried about `@staticmethod`, feel free to remove it, it is not used for any particular reason other that we didn't use any internal state before (aka `self`).

Reply:
hdfs is a static method. I cant use the self directly. Hence the rest of the logic

Reply:
Okay thanks.. I will remove the  staticmethod.